systest/backup/advanced-scenarios/127-Namespace/backup_test.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"net/http"
+	"testing"
+
+	e2eCommon "github.com/dgraph-io/dgraph/graphql/e2e/common"
+	utilsCommon "github.com/dgraph-io/dgraph/systest/backup/common"
+	"github.com/dgraph-io/dgraph/testutil"
+	"github.com/dgraph-io/dgraph/x"
+)
+
+var (
+	headerAlpha1Np0   = http.Header{}
+	headerAlpha1Np51  = http.Header{}
+	headerAlpha1Np130 = http.Header{}
+
+	headerAlpha2Np0   = http.Header{}
+	headerAlpha2Np51  = http.Header{}
+	headerAlpha2Np130 = http.Header{}
+)
+
+const (
+	accessJwtHeader = "X-Dgraph-AccessToken"
+	restoreLocation = "/data/backups/"
+	backupDst       = "/data/backups/"
+)
+
+func Test127PlusNamespaces(t *testing.T) {
+	jwtTokenAlpha1Np0 := testutil.GrootHttpLoginNamespace("http://"+testutil.SockAddrHttp+"/admin", 0).AccessJwt
+	headerAlpha1Np0.Set(accessJwtHeader, jwtTokenAlpha1Np0)
+	headerAlpha1Np0.Set("Content-Type", "application/json")
+
+	jwtTokenAlpha2Np0 := testutil.GrootHttpLoginNamespace("http://"+testutil.ContainerAddr("alpha2", 8080)+"/admin", 0).AccessJwt
+	headerAlpha2Np0.Set(accessJwtHeader, jwtTokenAlpha2Np0)
```

Comment:
a lot of this code can be wrapped inside a function and can be re-used.

Reply:
Resolved